lib/index.css

```diff
@@ -16,11 +16,151 @@
 
 @import url(lib/bike-rental.css);
 
+@import url(react-sliding-pane/dist/react-sliding-pane.css);
+
+
 /* Hide IE/Edge clear button in text input fields. */
 input[type=text]::-ms-clear {
     display: none;
 }
 
+/* New app menu */
+.app-menu-icon {
+  width: 21px;
+  height: 15px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  cursor: pointer;
+  position: absolute;
+  z-index: 10;
+  transition: all 1s ease;
+  top: 16px;
+}
+
+.menu-top-line, 
+.menu-middle-line, 
+.menu-bottom-line,
+.menu-left-x,
+.menu-right-x {
+  border-bottom: 3px solid #ffffff;
+  transition: all 0.5s ease;
+}
+
+.menu-left-x {
+  transform: rotate(45deg);
+  top: 7px;
+  position: relative;
+}
+
+.menu-right-x {
+  transform: rotate(-45deg);
+  bottom: 5px;
+  position: relative;
+}
+
+.slide-pane {
+  transition: transform 0.4s;
+}
+
+.slide-pane_from_left {
+  margin: 52px auto 0 0;
+}
+
+.slide-pane__header,
+.slide-pane__close {
+  display: none;
+}
+
+.slide-pane__content {
+  padding: 24px 0;
+}
+
+.app-menu {
+  margin: 0;
+  padding: 0;
+  font-size: 1.3rem;
+}
+
+.app-menu li {
+  padding: 1rem 2rem;
+  cursor: pointer;
+}
+
+.app-menu li img {
+  width: 25px;
+  margin-right: 2rem;
+}
+
+.app-menu li a {
+  color: inherit;
+  text-decoration: none;
+  outline: none;
+}
+
+.app-menu li a span,
+.expand-submenu-button span span {
+  font-size: 1.9rem;
+}
+
+.app-menu span:not(.expand-menu-chevron) {
+  margin-right: 2rem;
+}
+
+.menu-item:hover *,
+.app-menu li a span *:hover {
+  color: #4c97f5;
+}
+
+
+.expand-menu-chevron {
+  margin-right: -2rem;
+  justify-self: flex-end;
+}
+
+.dropdown-header {
+  font-size: inherit;
+  line-height: normal;
+  color: inherit;
+  white-space: nowrap;
+}
+
+.expand-submenu-button {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  margin: 0rem 0rem 0rem -0.5rem;
+  background: transparent;
+  border: none;
+  outline: none;
+  padding-bottom: 1rem;
+}
+
+.sub-menu-container {
+  border-top: 1px solid #cccccc;
+  padding-top: 1rem;
+}
+
+.sub-menu-container li a {
+    padding: 1rem 1rem 1rem 2rem;
+}
+
+.sub-menu-container li a span {
+  font-size: 1.3rem;
+  color: inherit;
+}
+
+.sub-menu-container li a span span {
+  font-size: 2rem !important;
+}
+
+.app-menu-separator {
+    border-bottom: 1px solid #cccccc;
+    margin-bottom: 1.3rem;
```

Comment:
If the padding is made tighter, I think this is where it also would have to be adjusted to keep the current (really good) spacing

Reply:
I've tested downgrading React locally and the menu still seems to work and the tests pass!